### ISSUE TITLE
[UI] Render different varieties of instance group capacities

### ISSUE THREAD

**Issue by beeankha** [ansible/awx#10673]
Instance Group Capacities

As the controller mesh is being fleshed out we anticipate that a instance group will have several different kinds of capacity that we should display.  Those capacities could be:

1. Total Capacity
2. Control Capacity or Execution Capacity
3. Use Capacity.

 
There could be some cases where there is both Control Capacity and Execution Capacity.  The list items are already quite crowded IMO, so displaying all these different types of capacity might be difficult without an expanded list view.  We should also display the different capacities in the details view.  If we do introduce an expanded row on this list we should also introduce the expand all functionality that we have on the jobs list, template list and project list.

API team members assigned to this work item are @AlanCoding, @beeankha, @jladdjr, @thenets.  

@trahman73 could you provide some thoughts about how to display these?

**Comment by trahman73** [comment]
@beeankha I agree with your solution of having an expanded list view as similar to our Jobs list view
https://tower-mockups.testing.ansible.com/patternfly/jobs/jobs-expand/

We would also want to add these to the Instance details view here:
https://tower-mockups.testing.ansible.com/patternfly/instance-groups/instance-groups-detail/

This is the UX flow that we already have in place so I feel this could work for your above requirements.

**Comment by AlanCoding** [comment]
Users might not really care about control capacity.

There is a specific scenario we were concerned about for control capacity: In a cluster with both control-only nodes and hybrid nodes, the hybrid nodes may be running large jobs. In those cases, jobs will push the nodes over-capacity and we would like to preference the control-only nodes to serve as job controllers because they have more free resources.

Maybe an expanded card would be an appropriate place to show multiple forms of capacity. Our first priority is to surface the execution capacity to the user.

**Comment by trahman73** [comment]
The rule we are trying to follow for expanded data in the main list view is that it needs to be important enough to be at a glance. Anything else should really be placed or repeated in the individual resource details view.

**Comment by AlanCoding** [comment]
I believe this issue was a duplicate with https://github.com/ansible/awx/issues/10767.

To recap, as per https://github.com/ansible/awx/issues/10830, it makes more sense to not allow putting control-only nodes inside of user-defined instance group.

yes, it's true that `controlplane` will have a `capacity` number which is not 100% available to jobs in cases. That would be unique to that one instance group, and adding a field to _every_ instance group doesn't sound very smart.

In other words, I think we're not going to do this, and I am going to close the issue to avoid any mixed messages.
